src/blocking/util.rs

```diff
@@ -12,17 +10,24 @@ use std::string::ToString;
 
 use super::oauth2::{SpotifyOAuth, TokenInfo};
 
+use getrandom::getrandom;
+
+static ALPHANUM: &[u8] =
```

Comment:
Shouldn't this be inside the `generate_random_string` function? That way we avoid unnecessary global variables.

Reply:
Yes, it makes sense.